spec/helpers/friends_helper_spec.rb

```diff
@@ -15,7 +13,7 @@
 
     it 'returns the last checkin address if it exists' do
       expect(helper.friends_device_last_checkin(device.checkins.first)).to match('Last available')
-      expect(helper.friends_device_last_checkin(device.checkins.first)).to match(device.checkins.first.fogged_city.to_s)
+      expect(helper.friends_device_last_checkin(device.checkins.first)).to match(device.checkins.first.city.to_s)
```

Comment:
Looks like I found a small bug here. B/c helper invokes methods that work with 'city' attribute, not fogged. Please, confirm.